File: Fact_Mult/Multi.js
/*
    Author : Pakshal Jain
    Date : 01/10/2022
    Program : To Calculate Multiplication of Factors of Entered Number
*/
//Function To Calculate Multiplication of factors of Entered Number
function Fact_Mult(No) {
    var i = 1;
    var Mult = 1;
    //Logic
    for (i = 1; i < No; i++) {
        if (No % i == 0) {
            Mult = Mult * i;
        }
    }
    return Mult;
}
//Main Function
function main() {
    var no = 56;
    var iret = 0;
    console.log("Jay Ganesh....");
    //Call To Fact_Mult Function
    iret = Fact_Mult(no);
    console.log("Multiplication of Factors of Entered Number is : " + iret);
}
//Call To Main Function
main();

File: Fact_Mult/Multi.py
"""
    Author : Pakshal Jain
    Date : 01/10/2022
    Program : To Calculate Mulitplication of Factor of Entered Number
"""

#Function To Calculate Multiplication of Factors of Entered Number
def Fact_Mult(No) :
    Mult = 1;

    #Logic 
    for i in range(1,No) :
        if(No % i == 0) :
            Mult = Mult * i;

    return Mult;

######################################################################################################

#Main Function
def main() :
    No = 0;
    iret = 0;

    print("Jay Ganesh...");
    print("Enter One Number");
    No = int(input());

    #Calling Fact_Mult Function
    iret = Fact_Mult(No);
    print("Multiplication of Factors of Entered Number is :",iret);

if __name__ == "__main__" :
    #Call To main Function
    main();